features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js

```diff
@@ -0,0 +1,204 @@
+        }
+
+
+        $("#refresh-policy").click(function () {
+            $('#policy-spinner').removeClass('hidden');
+            loadPolicyCompliance();
+        });
+
+        $("#refresh-operations").click(function () {
+            $('#operations-spinner').removeClass('hidden');
+            loadOperationsLog(true);
+        });
+
+    });
+
+    function loadOperationsLog(update) {
+        var operationsLog = $("#operations-log");
+        var deviceListingSrc = operationsLog.attr("src");
+        var deviceId = operationsLog.data("device-id");
+        var deviceType = operationsLog.data("device-type");
+
+        $.template("operations-log", deviceListingSrc, function (template) {
+            var serviceURL = "/devicemgt_admin/operations/" + deviceType + "/" + deviceId;
+
+            var successCallback = function (data) {
+                data = JSON.parse(data);
+                $('#operations-spinner').addClass('hidden');
+                var viewModel = {};
+                viewModel.operations = data;
+                if (data.length > 0) {
+                    var content = template(viewModel);
+                    if (!update) {
+                        $("#operations-log-container").html(content);
+                        $('#operations-log-table').datatables_extended();
+                    } else {
+                        $('#operations-log-table').dataTable().fnClearTable();
+                        for (var i = 0; i < data.length; i++) {
+                            var status;
+                            if (data[i].status == "COMPLETED") {
+                                status = "<span><i class='fw fw-ok icon-success'></i> Completed</span>";
+                            } else if (data[i].status == "PENDING") {
+                                status = "<span><i class='fw fw-warning icon-warning'></i> Pending</span>";
+                            } else if (data[i].status == "ERROR") {
+                                status = "<span><i class='fw fw-error icon-danger'></i> Error</span>";
+                            } else if (data[i].status == "IN_PROGRESS") {
+                                status = "<span><i class='fw fw-ok icon-warning'></i> In Progress</span>";
+                            }
+
+                            $('#operations-log-table').dataTable().fnAddData([
+                                                                                 data[i].code,
+                                                                                 status,
+                                                                                 data[i].createdTimeStamp
+                                                                             ]);
+                        }
+                    }
+                }
+
+            };
+            invokerUtil.get(serviceURL,
+                            successCallback, function (message) {
+                        console.log(message);
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/303/commits/051493db6a3fdb2e75cc7d1f48cb3ed407622312